.circleci/config.yml

```diff
@@ -52,7 +53,16 @@ jobs:
       - run: make test TEST_FLAGS="-race -timeout 60s"
       - run: make all
       - run: make test-docs
-      - run: make e2e
+      - run:
+          name: End-to-end Helm v2
+          command: E2E_KIND_CLUSTER_NUM=4 make e2e
+          environment:
+            HELM_VERSION: v2
+      - run:
+          name: End-to-end Helm v3
+          command: E2E_KIND_CLUSTER_NUM=4 make e2e
+          environment:
+            HELM_VERSION: v3
```

Comment:
Does it make sense to both pass the env variables through the shell and through `environment`? I would go for one of the two mechanisms

Reply:
I used the `environment` for the Helm version to highlight the difference between the two `run` items.

Reply:
alright